package.json

```diff
@@ -743,5 +744,5 @@
       "level>classic-level": false
     }
   },
-  "packageManager": "yarn@4.4.1"
+  "packageManager": "yarn@4.5.1"
```

Comment:
Update to yarn should not be needed I guess.

Reply:
Good catch Jyoti! Reverted this back